web_poet/fields.py

```diff
@@ -216,3 +225,86 @@ def _without_unsupported_field_names(
     if item_field_names is None:  # item_cls doesn't define field names upfront
         return field_names[:]
     return list(set(field_names) & set(item_field_names))
+
+
+@attrs.define
+class SelectFields:
+    """This is used as a dependency in :class:`~.ItemPage` to control which
+    fields to populate its returned item class.
+
+    You can also use this to enable some fields that were disabled by default
+    via the ``@field(disabled=True)`` decorator.
+
+    Some usage examples:
+
+    * ``SelectFields({"name": True})`` — select one field
+    * ``SelectFields({"name": False})`` — unselect one field
+    * ``SelectFields({"*": True})`` — select all fields
```

Comment:
Let's add a note about disabled fields: are they selected this way or not? E.g. "select all fields, including disabled"

Reply:
Speaking of `disabled` fields, are we good to go with this terminology? or continue considering renaming it as well as previously explored in https://github.com/scrapinghub/web-poet/pull/120#discussion_r1090424890.